sdk/src/main/java/com/sygic/travel/sdk/api/model/geo/Bounds.kt

```diff
@@ -17,4 +18,15 @@ internal class Bounds {
 
     @SerializedName("west")
     var west: Float = 0.toFloat()
+
```

Comment:
Generally, anemic model is considered as a bad practice, so it should be ok to put logic there, the question may be if this is the logic which should be here.